cfpb/consumerfinance.gov
# Paying for College - Non-first-year student features

This Pull Request adds features for non-first-year students. It also reformats some graphs and content.

## Changes
- Changes to school selection screen which allows a student to select their previous schooling
- Changes to debt calculation and inputs for current student debt
- Updates to graphs and content to improve design and clarity

## How to test this PR
1.  Pull down the branch and get it running
2. Check the various fields and user experiences of non-first-year students (see notes below)

## Notes and todos
- For details on how the user experience should work, please consult internal issues 356, 357, and 359.


## Checklist

### Front-end testing
#### Browser testing
Visually tested in the following supported browsers:

#### Accessibility

#### Other
